src/app/state/pairSelectorSlice.ts

```diff
@@ -143,6 +143,9 @@ export const pairSelectorSlice = createSlice({
         state.token2 = { ...state.token2, balance };
       }
     },
+    resetBalance: () => {
+      return initialState;
```

Comment:
Can you only reset the token1 and token2 balances? Otherwise the pair selection will be resettet too, which I don't think is what we want :)

e.g. either delete or set to undefined, but I'd prefere to restore to the default state, which is that the field is not present at all:
```js
resetBalances: (
      state: PairSelectorState
    ) => {
      delete state.token1.balance;
      delete state.token2.balance;
    },
```